src/dashboard/src/main/models.py

```diff
@@ -401,14 +401,23 @@ def __str__(self):
         )
 
 
-class UnitHiddenManager(models.Manager):
+class PackageManager(models.Manager):
     def is_hidden(self, uuid):
         """ Return True if the unit (SIP, Transfer) with uuid is hidden. """
         try:
             return self.get_queryset().get(uuid=uuid).hidden
         except:
             return False
 
+    def done(self, completed_before=None, include_failed=True):
+        statuses = [PACKAGE_STATUS_DONE, PACKAGE_STATUS_COMPLETED_SUCCESSFULLY]
+        if include_failed:
+            statuses.append(PACKAGE_STATUS_FAILED)
```

Comment:
It might be good to add `PACKAGE_STATUS_UNKNOWN` here as well for backward compatibility (maybe with a separate CLI flag if we want to be extra careful). Otherwise it's not currently possible to purge data from SIPs created before the `Package.status` field existed.

Reply:
I think that MCPServer is going to be restarted when the user is uprgrading Archivematica and at that point MCPServer will mark these packages as failed, but I still think it's a good idea just in case we're not predicting other scenarios.